rabbitmq/README.md

```diff
@@ -5,7 +5,9 @@ This demonstration describes how to create a RabbitMQ cluster in Openshift.
 ![RabbitMQ](images/RabbitMQ-logo.svg "RabbitMQ")
 
 ## Requirements
-This example is configured to use a `PersistentVolume` for storing cluster and message data. Thus it is a requirement that Openshift is configured to support [Persistent Volumes](https://docs.openshift.com/container-platform/latest/dev_guide/persistent_volumes.html) and that there are PVs with at least `ReadWriteOnce` (RWO) access available.
+1. This example is configured to use a `PersistentVolume` for storing cluster and message data. Thus it is a requirement that Openshift is configured to support [Persistent Volumes](https://docs.openshift.com/container-platform/latest/dev_guide/persistent_volumes.html) and that there are PVs with at least `ReadWriteOnce` (RWO) access available.
+
+2. This example is also using the [OpenShift Applier](https://github.com/redhat-cop/casl-ansible/tree/master/roles/openshift-applier) to build and deploy RabbitMQ. As a result you'll need to have [ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html).
```

Comment:
Somewhere in between this statement and below where you walk through the steps, you should let the reader know what the automation is about to build, especially the namespace its going to create.